tasks/Task.cpp

```diff
@@ -26,178 +26,128 @@ Task::~Task()
 
 bool Task::configureHook()
 {
-// - todo: use the method IsConnected();
-// - use TryGetTemperature(float *temperature) to get temperature;
-// - verify what is NavData, it seems to be a file that stores the navigation log, 
-//wich might be useful for offline data treatment. see PutNavData(const NavData &nav_data);
-// - See what is "orientation", it is gonna be implemented in the bvt_monitor, but maybe it's 
-//necessary to create a input port that will receive data from the rotation motor and set the
-//bvt orientation using PutOrientation and retrieve with GetOrientation;
-// - see what is EventMark class;
-
-/*try
-{
-if (son.SupportsSourceLevelControl())
-    son.SourceLevel(_source_level.get();
-}
-catch
-{
-
-}
-*/
     if (! TaskBase::configureHook())
         return false;
 
-    char DataFile[] = "/home/diego/Programas/rock/drivers/sonar_bvt_sdk/data/swimmer.son";
-
-    // Create a new BVTSonar Object
-    son = BVTSonar_Create();
-    if( son == NULL )
-    {
-        printf("BVTSonar_Create: failed\n");
-        return 1;
-    }
-
-    // Open the sonar
-    int ret;
-    ret = BVTSonar_Open(son, "FILE", DataFile);
-    if( ret != 0 )
-    {
-        printf("BVTSonar_Open: ret=%d\n", ret);
-        return 1;
-    }
-
-    // Make sure we have the right number of heads
-    int heads = -1;
-    BVTSonar_GetHeadCount(son, &heads);
-    printf("BVTSonar_GetHeadCount: %d\n", heads);
```

Comment:
Same. This is C++, do not use printf but std::cout or (better) the Rock logger (http://rock-robotics.org/stable/documentation/debugging/index.html)